Assets/PatchKit Patcher/Scripts/AppData/Remote/RemoteResourceDownloader.cs

```diff
@@ -139,6 +139,31 @@ private bool AreMetaAvailable()
             return _resource.HasMetaUrls();
         }
 
+        private bool ShouldUseTorrents()
+        {
+            if (!_useTorrents)
+            {
+                return false;
+            }
+
+            string forceHttpVal;
+            bool isForceHttpSet = EnvironmentInfo.TryReadEnvironmentVariable(
+                EnvironmentVariables.ForceHttp, out forceHttpVal);
+
+            if (!isForceHttpSet)
+            {
+                return true;
+            }
+
+            bool forceHttp;
+            if (Boolean.TryParse(forceHttpVal, out forceHttp))
```

Comment:
Does that accept `true` as well as `1` strings?

Reply:
According to [msdn](https://msdn.microsoft.com/pl-pl/library/system.boolean.tryparse(v=vs.110).aspx) It only accepts "true" or "false"